client/src/components/pages/Landing/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '../../common/Button';
+import { logo } from '../../assets/svgPaths';
+import './index.css';
+
+const LandingPage = () => {
+  return (
+    <div className="landing">
+      <div className="landing__logo">
+        <svg
+          width="188"
+          height="158"
+          viewBox="0 0 188 158"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path d={logo} fill="url(#paint0_linear)" />
```

Comment:
`logo` is undefined

Reply:
It is imported up in the same file from the svgPaths file and it displays the logo in the page

Reply:
I must be blind 🙄